app/gui/src/presenter/project.rs

```diff
@@ -171,9 +171,33 @@ impl Model {
 
     fn save_project_snapshot(&self) {
         let controller = self.controller.clone_ref();
+        let breadcrumbs = self.view.graph().model.breadcrumbs.clone_ref();
         executor::global::spawn(async move {
             if let Err(err) = controller.save_project_snapshot().await {
-                error!("Error while saving module: {err}");
+                error!("Error while saving project snapshot: {err}");
+            } else {
+                breadcrumbs.set_project_changed(false);
+            }
+        })
+    }
+
+    /// This is called on any module change. It should be followed by a call to
+    /// `check_project_changed` to verify that there is indeed a project change compared to the last
+    /// saved VCS snapshot.
+    fn assume_project_changed(&self) {
+        self.view.graph().model.breadcrumbs.set_project_changed(true);
+    }
```

Comment:
1. If it sohuld be followed by a call to something, why we dont create one function that does both?
2. The only occurence of this function is not followed by `check_project_changed`! At least I cant see it.
3. I dont understand the docs - they tell that we should call `check_project_changed` to "verify there is indeed a project change". Why is that? Why this function even exists if we need an additional verification? This is IMO strange logic here.